src/Input/Silk.NET.Input.Common/PublicAPI/netstandard2.1/PublicAPI.Shipped.txt

```diff
@@ -330,6 +330,10 @@ static Silk.NET.Input.GamepadExtensions.LeftBumper(this Silk.NET.Input.IGamepad!
 static Silk.NET.Input.GamepadExtensions.LeftStick(this Silk.NET.Input.IGamepad! gamepad) -> Silk.NET.Input.Button
 static Silk.NET.Input.GamepadExtensions.RightBumper(this Silk.NET.Input.IGamepad! gamepad) -> Silk.NET.Input.Button
 static Silk.NET.Input.GamepadExtensions.RightStick(this Silk.NET.Input.IGamepad! gamepad) -> Silk.NET.Input.Button
+static Silk.NET.Input.GamepadExtensions.LeftThumbstick(this Silk.NET.Input.IGamepad! gamepad) -> Silk.NET.Input.Thumbstick
+static Silk.NET.Input.GamepadExtensions.LeftThumbstickButton(this Silk.NET.Input.IGamepad! gamepad) -> Silk.NET.Input.Button
+static Silk.NET.Input.GamepadExtensions.RightThumbstick(this Silk.NET.Input.IGamepad! gamepad) -> Silk.NET.Input.Thumbstick
+static Silk.NET.Input.GamepadExtensions.RightThumbstickButton(this Silk.NET.Input.IGamepad! gamepad) -> Silk.NET.Input.Button
```

Comment:
Do not modify the shipped files, these are unshipped APIs

Reply:
Noted. I'm not really sure what these files are for and I wasn't able to locate any documentation so I just went with the flow 😅